web/src/containers/admin/AdminDashboard.js

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+
+import Icon, { faUserCog, faUserPlus } from '../../components/Icons';
+import Footer from '../../components/Footer';
+import Header from '../../components/Header';
+
+const AdminDashboard = () => {
+  return (
+    <Fragment>
+      <Header />
+      <div className="site-body ds-l-container">
+        <div className="ds-l-row">
+          <div className="ds-l-col--8 ds-u-margin-x--auto ds-u-padding-y--6">
```

Comment:
I'm not crazy about putting the padding in with utility classes (ideally, our spacing would be standardized), but I'm planning to move it once things start to look a bit more uniform.